### PR TITLE
gke and eks updates

### PR DESCRIPTION
**Pull request by vyankyGH**

### CODE REVIEW

File: pkg/nirmata/gke_clusterType.go [hunk 1]
```diff
@@ -72,6 +72,41 @@ func resourceGkeClusterType() *schema.Resource {
 					return
 				},
 			},
+			"location_type": {
+				Type:     schema.TypeString,
+				Required: true,
```
Comment (JimBugwadia) [inline pkg/nirmata/gke_clusterType.go:77]: can we default to Regional?

File: pkg/nirmata/gke_clusterType.go [hunk 2]
```diff
@@ -72,6 +72,41 @@ func resourceGkeClusterType() *schema.Resource {
...
+			"node_locations": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Required: true,
+			},
+			"enable_secrets_encryption": {
+				Type:     schema.TypeBool,
+				Required: true,
```
Comment (JimBugwadia) [inline pkg/nirmata/gke_clusterType.go:88]: can we default to `true`

File: pkg/nirmata/gke_clusterType.go [hunk 3]
```diff
@@ -72,6 +72,41 @@ func resourceGkeClusterType() *schema.Resource {
...
+			},
+			"enable_workload_identity": {
+				Type:     schema.TypeBool,
+				Required: true,
```
Comment (JimBugwadia) [inline pkg/nirmata/gke_clusterType.go:92]: can we default to `true`?

**Review by JimBugwadia — CHANGES_REQUESTED** [review]
please see comments. would be good if we can provide defaults, where they make sense.

File: pkg/nirmata/gke_clusterType.go (hunk 2 above)
Comment (vyankyGH) [reply pkg/nirmata/gke_clusterType.go:88]: IF we make default true, then it's mandatory to add **Secret Encryption Key**. and this value user have to specified.
So we kept default false.

File: pkg/nirmata/gke_clusterType.go (hunk 3 above)
Comment (vyankyGH) [reply pkg/nirmata/gke_clusterType.go:92]: IF we make default true, then it's mandatory to add **Workload Pool**. and this value user have to specified.
So we kept default false.

Comment (JimBugwadia) [reply pkg/nirmata/gke_clusterType.go]: ok

File: pkg/nirmata/gke_clusterType.go (hunk 2 above)
Comment (JimBugwadia) [reply pkg/nirmata/gke_clusterType.go]: ok

**Review by JimBugwadia — CHANGES_REQUESTED** [review]
please resolve conflicts

**Pull request merged**